components/cdc/src/endpoint.rs

```diff
@@ -626,11 +646,36 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
                             region_id,
                             err: Error::Request(e.into()),
                         };
+                        tx.clone().send(Either::Right(region_id)).wait().unwrap();
```

Comment:
`wait` may block the worker.

Reply:
The capacity of channel is set to `region_count` so ideally this `wait` would be not blocked